components/header

```diff
@@ -0,0 +1,19 @@
+import React, { Component } from 'react';
```

Comment:
You don't need to import the entire react package so you can just keep it as `import { Component } from 'react';`